src/metabase/public_settings.clj

```diff
@@ -816,6 +816,48 @@ See [fonts](../configuring-metabase/fonts.md).")
                     ;; frontend should set this value to `true` after the modal has been shown once
                     v))))
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; v50 Permissions Tutorial settings
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Bryan (7/8/24): These 2 settings are meant to mitigate any confusion over the new and improved permissions format
+;; We switched to in v50 for users who were around proir to v50. We never show the banner or modal on instances
+;; created after v50.
+;; We should come back around and delete them in a few months.
```

Comment:
Should we actually _just_ merge this to the `release-x.50.x` branch but leave it out of master so that it's not in 51? Might be a product question for how long we want to leave this in. I guess some folks might skip 50 and upgrade straight to 51 from an earlier version so we could keep it in a few releases.

Reply:
I think I agree with what you mean, which is we should merge it to master, backport it, and revisit its existance in master in a couple months. I'll update the comment to make that clear